packages/fxa-settings/src/pages/Signin/SigninTotpCode/en.ftl

```diff
@@ -17,3 +17,5 @@ signin-totp-code-other-account-link = Use a different account
 signin-totp-code-recovery-code-link = Trouble entering code?
 # Error displayed in a tooltip when the form is submitted without a code
 signin-totp-code-required-error = Authentication code required
+# Briefly displayed when totp code has been successfully validated
+signin-totp-code-success = Success!
```

Comment:
Do we display this in Backbone? If we don't I wouldn't add it since a redirect happens right after and that can be jarring for folks that use a screenreader (e.g. seeing a flash of something and then being redirected without being able to see what it was) but I haven't ran through this locally just yet so probably carry on, lol.

Reply:
But for folks not using a screen reader, the intermediate step, where one might be waiting for navigation would have no feedback concerning the previous action. That seems much worse to me...

Reply:
Out of scope here, but I think the "most correct" thing might be to have a success state in the alert bar when someone's logged in and redirected to Settings, probably also for `signin_token_code`. That way the screenreader vs non-screenreader experience is the same (which is important). When I ran through this locally, I'm not even sure I saw the "Success" message before I was redirected. Fine with me to leave this, but I know UX has been looking at the sign in/sign up states recently, so it could be good to create a UX ticket for this to get what is most ideal for when we start on that work. 🤷‍♀️

Reply:
I've removed it... Even simulating a really slow connection, I couldn't see the message...